charts/lnd/Chart.yaml

```diff
@@ -16,8 +16,10 @@ engine: gotpl
 dependencies:
   - name: loop
     version: x.x.x
-  - name: rtl
+    condition: loop.enabled
```

Comment:
didn't know we could do that. is that a new feature of helm?

Reply:
Afaik it was there in helm v2 also, but the way to achieve it was a bit different. This was what I was referring to when we had the discussion over subchart enabling/disabling.